config/loader.go

```diff
@@ -75,8 +104,8 @@ func loadConfig(cfg interface{}, fs afero.Fs, dirPath string) error {
 
 	opts := []config.LoaderOption{
 		config.WithViper(v),
-		config.WithName(FileName),
-		config.WithType(FileExtension),
+		config.WithName(".optimus"),
```

Comment:
Will this function be deleted later ?

Reply:
all legacy function or struct related to `config.Optimus` will be deleted. I will put a comment on these to avoid confusion